### DISCUSSION TITLE
Récupérer les coordonnées d'un objet

### DISCUSSION THREAD

**Discussion by MarlBnc** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#77] [Q&A] [answered]
Bonjour !

Je me demandais ou récupérer les coordonnées des objets placés par Mapping ? Au château Marius, un deuxième frigo a été installé mais ne fonctionne pas. Je suis donc allé modifier les Storages " SOZ-FiveM-Server/resources/[soz]/soz-inventory/storages/food.lua " pour ajouter un second frigo, j'ai essayé de tâtonner avec les coordonnées pour placer la zone de b-target du frigo mais c'est compliqué

Y'a t-il une façon simple de les avoir et ou de visualiser les zones définies par les objets ?

Merci

**Comment by Laikker** [comment, marked as answer]
Salut,

oui tu peux juste ajouter debug=true dans la config de ton storage, par exemple:
`
Config.Storages["food_fridge"] = {
    label = "Frigo Château Marius",
    type = "fridge",
    owner = "food",
    position = vector3(-1878.76, 2066.86, 141.02),
    size = vector2(0.5, 1.5),
    offsetUpZ = 2.0,
    heading = 251.1,
    debug = true,
}
`

![image](https://user-images.githubusercontent.com/120526954/215297181-675e3665-b6db-47aa-8dcb-28d85bfc48d4.png)
